python/graphscope/nx/tests/classes/test_graph.py

```diff
@@ -358,7 +369,10 @@ def setup_method(self):
 
     def test_correct_edges(self):
         """Tests that the subgraph has the correct edges."""
-        assert [(0, 1, "edge01"), (4, 3, "edge34")] == sorted(self.H.edges(data="name"))
+        assert sorted(self.H.edges(data="name")) in (
+            [(1, 0, "edge01"), (4, 3, "edge34")],
+            [(0, 1, "edge01"), (4, 3, "edge34")],
```

Comment:
H is undirected graph and `(3, 4)` and `(4, 3)` are the same edge.
num_workers=1 and num_workers=2 would get different layout of edges since the order of node iterations is totally different.